src/core/Notification/Notification.baseStyles.ts

```diff
@@ -11,6 +11,7 @@ export const baseStyles = (theme: SuomifiTheme) => css`
   display: block;
   justify-content: space-between;
   padding-bottom: 10px;
+  background-color: ${theme.colors.whiteBase};
```

Comment:
Not quite sure if this should be inside the main class or not. Could this get overridden too easily by accident?

Reply:
Hmm.. not quite sure but maybe under the `fi-notification` would be better place.

And if we decide to do that then same change is to be done with Toast.

Reply:
I moved the styles inside the main classes. There were no styles inside the main class in Toast, so created the selector as well.